hivemind/optim/optimizer.py

```diff
@@ -618,6 +618,12 @@ def _load_averaged_gradients_into_optimizer_(self):
 
         self.grad_averager.notify_used_averaged_gradients()
 
+    def _load_local_gradients_into_optimizer(self):
+        """Fallback to using local gradients in the optimizer (instead of averaged gradients)"""
+        logger.log(self.status_loglevel, f"Proceeding with local gradients")
```

Comment:
Please add a comment that this can be optimized in case of one peer (if we'd ever need to optimize this case).

Reply:
just did it